utils/cliutils/commandsflags.go

```diff
@@ -1283,6 +1284,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  RequirementsFile,
 		Usage: "[Optional] [Pip] Defines pip requirements file name. For example: 'requirements.txt' ` `",
 	},
+	WithFixVersion: cli.BoolFlag{
+		Name:  WithFixVersion,
+		Usage: "[Optional] Set to true if you wish to display the audit results with fix version only ` `",
+	},
+	MinSeverity: cli.StringFlag{
+		Name:  MinSeverity,
+		Usage: "[Optional] Set the minimum severity to display results and filter the rest ` `",
```

Comment:
Set the minimum severity to display results and filter the rest
-->
Set the minimum severity of issues to display. Set one following values are accepted: Low, Medium, High or Critical